domain/learningandworkprogress/src/main/kotlin/uk/gov/justice/digital/hmpps/domain/learningandworkprogress/review/service/ReviewService.kt

```diff
@@ -131,14 +167,27 @@ class ReviewService(
     }
   }
 
+  /**
+   * Creates and returns the prisoner's initial [ReviewSchedule].
+   * Returns null if the prisoner has less than 3 months to serve, in which case no Review is necessary and one is not
+   * scheduled for them.
+   *
+   * Only certain sentence types are supported to be able to create a [ReviewSchedule], and of those SENTENCED and
+   * RECALL require the prisoner to have a release date. Throws a subclass of [InvalidReviewScheduleException] if these
+   * validation rules are not met.
+   */
   fun createInitialReviewSchedule(createInitialReviewScheduleDto: CreateInitialReviewScheduleDto): ReviewSchedule? {
-    // Check for an existing active review schedule
-    val existingReviewSchedule = runCatching {
-      getActiveReviewScheduleForPrisoner(createInitialReviewScheduleDto.prisonNumber)
-    }.getOrNull()
+    with(createInitialReviewScheduleDto) {
+      validateSentenceTypeAndReleaseDate(prisonNumber, prisonerSentenceType, prisonerReleaseDate)
+
+      // Check for an existing active review schedule
+      val existingReviewSchedule = runCatching {
+        getActiveReviewScheduleForPrisoner(prisonNumber)
+      }.getOrNull()
 
-    if (existingReviewSchedule != null) {
-      throw RuntimeException("An active review schedule already exists for prisoner ${createInitialReviewScheduleDto.prisonNumber}.")
+      if (existingReviewSchedule != null) {
+        throw ActiveReviewScheduleAlreadyExistsException(prisonNumber)
```

Comment:
Took the opportinity to throw a better exception here as well (this exception already exists and does exactly what it says on the tin 😁  )